frontend/src/stores/colors/NodeColorStore.ts

```diff
@@ -0,0 +1,10 @@
+import { Node } from '../../shared/entities';
+import BaseEntityColorStore from './BaseEntityColorStore';
+
+type NodeTypes = Node['types'];
+
+export default class NodeColorStore extends BaseEntityColorStore<NodeTypes> {
+  protected getTypeOfEntity(entityTypeId: NodeTypes): string {
+    return entityTypeId.sort((a, b) => a.localeCompare(b)).join(' ');
```

Comment:
Like in the refactoring, move this array `sort` operation to a separate statement, as `sort()` does alter the source array.

Other than that, LGTM.

Reply:
I created a copy of the original array so that sort does only alter the copied array and i still have method chaining.